test/pathforaSpec.js

```diff
@@ -105,6 +105,59 @@ describe('Pathfora', function () {
     jasmine.Ajax.uninstall();
   });
 
+  it('bonk bnok bonok', function (done) {
```

Comment:
hahahahahahaahahaha nice. i was struggling to get things to work, apparently i forgot to fix that once i got it straightened out :)